.gitignore

```diff
@@ -0,0 +1,4 @@
+image_1.png
+image_2.png
+ic_menu.png
```

Comment:
- [ ] Kindly remove the images' names from `.gitignore` file, because your design is broken now.

![image](https://user-images.githubusercontent.com/35535130/158730288-907976f5-1b56-47d1-b9e0-220e4e95ee39.png)

Reply:
resolved